app/hotels/src/GraphQLSanitizers.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import moment from 'moment';
+import { moment } from '@kiwicom/react-native-app-shared';
```

Comment:
@tbergq I think it's quite a good idea to create completely new packages if it makes sense ([not sure if our shared package is the right place](https://github.com/kiwicom/react-native-app/blob/master/app/shared/README.md)). What do you think?

Reply:
I think that sounds like a goodt idea, naming? just `@kiwicom/react-native-app-moment`?

Reply:
I actually wanted to hide the `moment` implementation detail. So I would maybe consider renaming `-translations` to `*-localization` and add it there with exports of just `DateFormatter` instead of `moment`? Something like that sounds good to me.

Reply:
Ok, adding it to `localization` makes sense. Not sure about naming it `DateFormatter`, We are basically just renaming `moment`. I think it makes sense when new people look at the code, that they actually understand at once that we are using `moment`